skills/synthesis/references/troubleshooting.md

```
@@ -0,0 +1,114 @@
+
+## Handling PR Feedback
+
+If the user receives PR review comments:
+
+1. Offer to address feedback:
+   ```typescript
+   Skill({ skill: "delegate", args: "--pr-fixes [PR_URL]" })
+   ```
+
+2. Delegate reads PR comments via GitHub MCP:
+   ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)